python/tvm/relay/frontend/onnx.py

```diff
@@ -879,6 +879,79 @@ def _get_convert_map(opset):
         'Shape': Shape.get_converter(opset),
     }
 
+supported_ops = set([
```

Comment:
I am thinking defining the supported_ops set is redundant.
 checking op_name in identity_list or op_name in convert_map or op_name = 'Constant' should be enough